integration/commands_administration_test.go

```diff
@@ -876,8 +876,6 @@ func TestCommandsAdministrationServerStatusMetrics(t *testing.T) {
 	} {
 		name, tc := name, tc
 		t.Run(name, func(t *testing.T) {
-			t.Parallel()
```

Comment:
Why did you remove it?

Reply:
I believe that's the reason why pg tests fail sometimes. `serverStatus` is the command which is not related to specific collection, so running these test cases in parallel makes it possible to read values in "wrong" order

Reply:
But what part of the test expects a particular order?

Reply:
Nevermind, now looking at tests it looks that they are order agnostic